x-pack/plugin/core/template-resources/src/main/resources/idp/saml-service-provider-template.json

```diff
@@ -13,10 +13,12 @@
     "index.priority": 10,
     "index.format": 1
   },
+  "version": ${idp.template.version},
```

Comment:
not related to the code ...but probably worth a quick spot check with the production IDP to ensure no one has manually updated the template since this will overwrite any of those changes up on install.